lib/server.js

```diff
@@ -3,13 +3,23 @@
 const express = require('express');
 const app = express();
 
-let db = require('./db.js');
+let db = require('../__data__/db.js');
 
 app.use(express.json());
 
```

Comment:
missing logger middleware, message middleware, 404 and 500 error middleware, and isValid middleware